cogs/utility.py

```diff
@@ -1432,6 +1440,188 @@ async def permissions_override(self, ctx, command_name: str.lower, *, level_name
             )
         return await ctx.send(embed=embed)
 
+    async def _bulk_override_flow(self, ctx):
+        embed = discord.Embed(
+            title="Bulk Override",
+            description=(
+                "Please list the commands you want to override. "
```

Comment:
Missing a \n at the end here
```suggestion
                "Please list the commands you want to override.\n"
```